mne/epochs.py

```diff
@@ -662,7 +662,7 @@ def __next__(self, *args, **kwargs):
     def __hash__(self):
         if not self.preload:
             raise RuntimeError('Cannot hash epochs unless preloaded')
-        return object_hash(dict(info=self.info, data=self._data))
+        return object_hash(dict(info=hash(self.info), data=self._data))
```

Comment:
It's going to be difficult to make the new hashes equal the old ones. It would require passing the hashlib object around, which would make things a bit messy. Maybe it's reasonable that an update to MNE can break existing hashes. For example, a field added to the `Info` object, as we have done in the past, will break hashes as well.